app/controllers/users_controller.rb

```diff
@@ -1,6 +1,15 @@
 # frozen_string_literal: true
 
 class UsersController < ApplicationController
+  before_action :logged_in_user, only: %i[edit update destroy]
```

Comment:
```suggestion
  before_action :logged_in_user, only: %i(edit update destroy)
```
tròn chứ sao vuông em nhỉ
thế show và index ko cần login à cu?

tìm hiểu thêm về `except` với `only` để dùng nhá, chỗ này đến 5 action thì dùng except sẽ ok hơn - cụ thể là kiệm code hơn